sidebars.js

```diff
@@ -17,7 +17,7 @@ const sidebars = {
         id: "introduction/introduction",
       },
       items: [
-        "introduction/what-is-secure-workforce",
+        "introduction/get-started-with-bi",
```

Comment:
Also will need to update this "getting-started". Since we are changing the path to this page, we will need to update the link to it in `docs/introduction` line 45.